kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer/freshness/Burrow.java

```diff
@@ -118,9 +118,15 @@ private HttpUrl address(String... paths) {
 
   public List<String> getClusterBootstrapServers(String cluster) throws IOException {
     final Map<String, Object> response = this.request(cluster);
-    final Map<String, Object> clusterDetail = (Map<String, Object>) response.get("cluster");
-    final Map<String, Object> module = (Map<String, Object>) clusterDetail.get("module");
-    return (List<String>) module.get("servers");
+    final Map<String, Object> module = (Map<String, Object>) response.get("module");
+    if (module == null) {
+      throw new IOException("response doesn't contain `module`: Response" + response);
```

Comment:
replace with a https://guava.dev/releases/19.0/api/docs/com/google/common/base/Preconditions.html ?

note that we uses shaded version of guava.

Reply:
done